reana.yaml

```diff
@@ -0,0 +1,18 @@
+version: 0.9.3
+inputs:
+  files:
+    - codes/analysis.py
```

Comment:
The directory should be named just `code`.

But, considering that Dask provides also workflow, so to speak, and that we don't have any other input files or data files, we could simply hos the sole `analysis.py` file in the root directory.